samples/sampleLib.py

```diff
@@ -0,0 +1,58 @@
+import json
+import vm_automation
+
+def checkForProcess(vmObject, processName):
+    vmObject.updateProcList()
+    if processName in ' '.join(vmObject.procList):
+        return True
+    else:
+        return False
+
+def getVmObjectFromName(vmServer, vmName):
+    vmServer.enumerateVms()
+    for vm in vmServer.vmList:
+        if vmName == vm.vmName:
+            return vm
+    return None
+
+def loadJsonFile(fileName):
+    try:
```

Comment:
Just opinion here but this seems a little more simple.

```
try:
    with open(fileName) as fileObject:
        fileDic = json.load(fileObject)
        return fileDic
except Exception as e:
    print ("UNABLE TO LOAD DICTIONARY FROM: " + str(fileName) + "\n" + str(e))
    return None
```

Reply:
Honestly, as finicky as the json reader is, I try and maintain distinct errors for file errors and json processing errors.  I know that it will print the error here, but splitting it has always been my preference.